app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -227,16 +227,17 @@ class CatalogueController @Inject()(
               TeamChartData.deploymentStability(team.name, teamIndicators.map(_.stability)),
               umpMyTeamsPageUrl(team.name),
               leakDetectionService.teamHasLeaks(team, reposWithLeaks),
-              leakDetectionService.hasLeaks(reposWithLeaks)
+              leakDetectionService.hasLeaks(reposWithLeaks),
```

Comment:
What do you think about changing the `service_info` view by creating an @Inject annotated args list and moving some of the existing args into that? I'm guessing it could de-clutter the controller's args and reduce the number of args we have to pass to `service_info`.